docs-en/chapter_data_structure/number_encoding.md

```diff
@@ -2,11 +2,11 @@
 
 !!! note
 
-    In this book, chapters marked with an * symbol are optional reads. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
+    In this book, chapters marked with an * symbol are optional to read. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
 
 ## Integer Encoding
 
-In the table from the previous section, we noticed that all integer types can represent one more negative number than positive numbers, such as the `byte` range of $[-128, 127]$. This phenomenon, somewhat counterintuitive, is rooted in the concepts of sign-magnitude, one's complement, and two's complement encoding.
+In the table from the previous section, we noticed that all integer types can represent one more negative number than positive numbers, such as the `byte` range of $[-128, 127]$. This phenomenon seems counterintuitive, and its underlying reason involves knowledge of sign-magnitude, one's complement, and two's complement encoding.
 
 Firstly, it's important to note that **numbers are stored in computers using the two's complement form**. Before analyzing why this is the case, let's define these three encoding methods:
```

Comment:
"Firstly, it's important to note that **numbers are stored in computers using the two's complement form**. Before analyzing why this is the case, let's define these three encoding methods:" can be changed to ->
"Firstly, it's important to note that **numbers are stored in computers using the 'complement' form**. Before delving into the reasons behind this choice, let's define these three encoding methods:"

Reply:
Here I think the original expression is simpler and easier to understand, so I don't want to modify it. But thank you again for providing a new translation idea.